rails.md

```diff
@@ -6,28 +6,37 @@ Answer the following questions. First, without external resources. Challenge you
 1. MVC (Model View Controller) is a pattern for the architecture of a software program. Give a brief description of each component and describe how Ruby on Rails handles MVC.
 
   Your answer:
+    Model:  manages data of application
+    View: what the user sees
+    Controller:  in between view and model
 
   Researched answer:
-
-
+    Model: contains data for application. They are Ruby classes and talk to the database, store and validate data, perform the business logic. Can be reused as they have no knowledge of user interfaces.
+    View:  generates the user interface which presents data to the user.
+           passive
+           many views can access the same model for different reasons.
+    Controller:  receives events from outside world.
+                 interacts with model.
+                 displays appropriate view to user.
+    MVC works together.
 
 2. Using the information given, fill in the blanks to complete the steps for creating a new view in a Rails application.
 
-  Step 1: Create the _________ in the file config/routes.rb
+  Step 1: Create the route_________ in the file config/routes.rb
   ```
   get '/about' => 'statics#about'
   ```
 
-  Step 2: Create the ____________ in the file _____________
+  Step 2: Create the Controller____________ in the file controller_____________
   ```
-  class ____________ < ApplicationController
-    def ___________
-      render ________________
+  class Controller ____________ < ApplicationController
+    def view___________
+      render html: view________________
     end
   end
   ```
```

Comment:
class StaticsController < ApplicationController
  def about
    render "about.html.erb"
  end
end
I'm getting all this information for this controller method from the example route above. The route tells me the controller is called statics and the method the route is calling is named "about". If the method is named "about" I know  the view will have the same name.